### ISSUE TITLE
Rule proposal: newline between same level html elements

### ISSUE THREAD

**Issue by kailniris** [vuejs/eslint-plugin-vue#741]
**Please describe what the rule should do:**

Enforce newline between same level HTML elements.

**What category should the rule belong to?**

- [x] Enforces code style
- [ ] Warns about a potential error
- [ ] Suggests an alternate way of doing something
- [ ] Other (please specify:)

**Provide 2-3 code examples that this rule should warn about:**
```vue
<template>
  <div>content</div>

  <CustomElement />

  <div>
    <div>content</div>

    <CustomElement />
  </div>   
</template>
```

**Additional context**

I think this really helps code readability. Maybe for some reasons this can be a bad practice. Ether way there shell be a rule to enforce or prevents this.

**Comment by FloEdelmann** [comment]
This is now possible with the [`padding-line-between-tags`](https://eslint.vuejs.org/rules/padding-line-between-tags.html) rule.
